### PR TITLE
cadence 0.37.0; update livecheck

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by ZhongRuoyu — COMMENTED** [review]
Not marked as the latest release yet: https://github.com/onflow/cadence/releases

**Review by SMillerDev — DISMISSED** [review]

**Comment by ZhongRuoyu** [comment]
@SMillerDev maybe you meant to "comment" rather than to "approve"? It’s not yet announced, so it might be a bit early to merge this.

**Comment by SMillerDev** [comment]
Yeah, missed that comment. We should adjust the livecheck to check the announcement then though

**Comment by ZhongRuoyu** [comment]
Also filed an issue to confirm the status of release 0.37.0: https://github.com/onflow/cadence/issues/2398

File: Formula/cadence.rb [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-  sha256 "6f14e98eade0c9bed77ca79bb00b2706fea1ccf83a06a1fbd739d98f96af038f"
+  url "https://github.com/onflow/cadence/archive/v0.37.0.tar.gz"
+  sha256 "73d8d9f1bcd102c9726abf7fcbb892d951aaf556b7e005e900542f3d37b8ad49"
   license "Apache-2.0"
   head "https://github.com/onflow/cadence.git", branch: "master"
 
   livecheck do
     url :stable
     regex(/^v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_latest
```
[the remark this answers is not in the document]
Comment (ZhongRuoyu) [reply Formula/cadence.rb:12]: I see. Thanks @samford!

**Comment by ZhongRuoyu** [comment]
Upstream has announced the release: https://github.com/onflow/cadence/releases/tag/v0.37.0

**Comment by chenrui333** [comment]
🚀

**Review by ZhongRuoyu — APPROVED** [review]

**Pull request closed without merging**